### DISCUSSION TITLE
How can I get a long click, like mousedown - wait 1 sec - mouse up

### DISCUSSION THREAD

**Discussion by ellieot** [seleniumbase/SeleniumBase#1857] [Q&A] [answered]
I cant use javascript to solve this, but is there better way using sbase?

**Comment by mdmintz** [comment, marked as answer]
You can use the `ActionChains` `click_and_hold()` method to do that:
https://www.selenium.dev/selenium/docs/api/py/webdriver/selenium.webdriver.common.action_chains.html#selenium.webdriver.common.action_chains.ActionChains.click_and_hold

Then use the `ActionChains` `release()` method when done holding:
https://www.selenium.dev/selenium/docs/api/py/webdriver/selenium.webdriver.common.action_chains.html#selenium.webdriver.common.action_chains.ActionChains.release

More info: https://www.selenium.dev/selenium/docs/api/py/_modules/selenium/webdriver/common/action_chains.html#ActionChains.click_and_hold

**Reply by ellieot** [reply]
thank you, but how can I use selenium method in seleniumbase, I cant find a way to do this. Sorry for bring this easy quest to you.

**Reply by mdmintz** [reply]
In a SeleniumBase test, for example:

```python
element = self.find_element("input")
from selenium.webdriver.common.action_chains import ActionChains
actions = ActionChains(self.driver)
actions.click_and_hold(element).pause(1).release(element).perform()
```

It's mostly the same as raw Selenium, but with a few changes, such as passing ``self.driver`` instead of ``driver``, for example.
